webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -116,28 +133,33 @@ export const authSlice = createSlice({
   reducers: {},
   extraReducers: (builder) => {
     builder
-      .addCase(login.fulfilled, (status, action) => {
-        if (action.payload != null) {
-          status.isAuthenticated = true;
-        }
+      .addCase(login.fulfilled, (state, { payload }) => {
+        state.sessionId = payload.access_token;
+      })
+      .addCase(login.rejected, (state) => {
+        state.sessionId = '';
+      })
+      .addCase(logout.fulfilled, (state) => {
+        state.isInAnonymousMode = true;
```

Comment:
Shouldn’t it be?
```suggestion
        state.isInAnonymousMode = false;
```

Reply:
The initial state is `true`. It'll only become `false` if the call to `/security_enabled` api is successful.